spec/definitions.yaml

```diff
@@ -928,14 +928,27 @@ definitions:
         description: |
           Attributes specific to the AWS provider
         properties:
+          instance_distribution:
+            description: |
+              Attributes defining the instance distribution in a node pool.
+            type: object
+            properties:
+              on_demand_base_capacity:
+                description: |
+                  Base capacity of on demand machines.
```

Comment:
The description should explain what exactly the number would mean. Is it a percentage or a number of machines?